api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto

```diff
@@ -56,4 +57,10 @@ message LeastRequest {
   // Configuration for slow start mode.
   // If this configuration is not set, slow start will not be not enabled.
   config.cluster.v3.Cluster.SlowStartConfig slow_start_config = 3;
+
+  oneof locality_config_specifier {
```

Comment:
Looks like we're repeating this `oneof` in 3 different places.  Would it make sense to define a new message in common.proto that contains this oneof, and then just have a single field of that new message type in each of these three places?

Reply:
Sounds great.